llvm/CMakeLists.txt

```diff
@@ -28,6 +28,15 @@ if(NOT DEFINED LLVM_VERSION_SUFFIX)
   set(LLVM_VERSION_SUFFIX git)
 endif()
 
+# Export a few LLVM version identifiers for users who use LLVM as a subdir.
```

Comment:
This seems a bit ad-hoc compared to every else that could be exported?

You mention parity with FindPackage(LLVM) in CMake: what else does  FindPackage exports?

Reply:
i don't know where the canonical source is, but here is some copy of FindLLVM.cmake with comment answering that: <https://github.com/ldc-developers/ldc/blob/master/cmake/Modules/FindLLVM.cmake#L7-L25>

I'm not sure I want to export all these variables I had never heard of, and conversely, this doesn't mention `LLVM_VERSION_SUFFIX`.

How about I just restrict this PR to just `LLVM_VERSION_MAJOR` ?

Reply:
Done.

Reply:
And so the answer to "why export this and not all the other variables" is that this one variable determines clang internal paths such as the builtin-headers include path.